include/tlog/errs.h

```diff
@@ -141,4 +141,36 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+#define TLOG_ERRS_RAISES(_perrs, _msg) \
+     do {                                       \
+         struct tlog_errs **__perrs = (_perrs); \
+         tlog_errs_pushs(__perrs, _msg);        \
+         goto cleanup;                          \
+     } while (0)
+
+#define TLOG_ERRS_RAISECS(_grc, _perrs, _msg) \
```

Comment:
I think we need to make all the macros use format strings (i.e. `tlog_errs_pushf`, for simplicity). Also, please name the format string argument `_fmt`, not `_msg`, since that's what it is, similarly to the way it is in `tlog_errs_pushf`